python/templates/error.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>500 Error</title>
```

Comment:
this head doesnt include the stylesheet (maybe thats intentional)

Reply:
Not really intentional :)

Reply:
actually modified error.html to mimic exactly what is rendered by flask's 500 template -- just modified the text. No stylesheet is included in flask's template either.